### DISCUSSION TITLE
MSG Makefile error in OMNET - 5.6.2

### DISCUSSION THREAD

**Discussion by soha-18** [omnetpp/omnetpp#1046] [Q&A]
Hello,
I am facing an error while creating MSG file. The error is as below:|
Dspr.msg:15: Error: syntax error
make[1]: *** [Makefile:109: Dspr_m.h] Error 1

Dspr.msg file is attached below.
[dspr.msg](https://github.com/omnetpp/omnetpp/files/12490904/dspr.txt)

Can anyone suggest the possible reason behind this error?
```
import inet.common.INETDefs;
import inet.common.TlvOptions;
import inet.common.geometry.Geometry;
import inet.common.packet.chunk.Chunk;
import inet.networklayer.common.L3Address;


class DsprOption extends TlvOptionBase {
    Coord destinationPosition;
    L3Address senderAddress;
    int hops;
    L3Address nextHopAddress;
}
```

**Comment by rhornig** [comment]
This is most likely not an OMNeT++ related issue so I'm converting it to a discussion.

**Reply by rhornig** [reply]
What is the INET version you are using? The file looks content looks to be fine.

It is also suspicious that the error is reported in line 15 while the whole file contains only 14 lines. Are you sure you are providing the right file?
